src/toil/batchSystems/kubernetes.py

```diff
@@ -497,47 +489,19 @@ def _ourJobObjects(self, onlySucceeded=False, limit=None):
         # https://github.com/kubernetes-client/python/issues/953. For now we
         # will just throw an error if we don't get to the end of the list in
         # time.
-        token = None
-        
-        # Do our own limiting since we need to apply a filter that the server
-        # can't.
-        seen = 0
-        
-        # TODO: We ought to label our jobs by owning Toil workflow so we can
-        # look them up instead of filtering down later.
         
-        while True:
-            # We can't just pass e.g. a None continue token when there isn't
-            # one, because the Kubernetes module reads its kwargs dict and
-            # cares about presence/absence. So we build a dict to send.
-            kwargs = {}
-            if onlySucceeded:
-                # Check only successful jobs.
-                # Note that for selectors it is "successful" while for the
-                # actual object field it is "succeeded".
-                kwargs['field_selector'] = 'status.successful==1'
-            if token is not None:
-                kwargs['_continue'] = token
-            
-            results = self._try_kubernetes(self._api('batch').list_namespaced_job, self.namespace, **kwargs)
-            
-            for job in results.items:
-                if self._isJobOurs(job):
-                    # This job belongs to us
-                    yield job
-                    
-                    # Don't go over the limit
-                    seen += 1
-                    if limit is not None and seen >= limit:
-                        return
-                    
-            # Remember the continuation token, if any
-            token = getattr(results.metadata, 'continue', None)
+        if onlySucceeded: 
+            return self._try_kubernetes(self._api('batch').list_namespaced_job, self.namespace, 
+                                            label_selector="toil_run={}".format(self.runID), field_selector="status.successful==1").items
+ 
+        return self._try_kubernetes(self._api('batch').list_namespaced_job, self.namespace, 
+                                            label_selector="toil_run={}".format(self.runID)).items
+       
+    
+    def _getOurPodObjectList(self):
```

Comment:
This ought to have a docstring. Also, why does this one have `get` in it when `_ourJobObjectList` doesn't? They should match.